docs/docs/widgets/button-group.md

```diff
@@ -56,7 +54,7 @@ There are currently no CSA (Component-Specific Actions) implemented to regulate
 
 | <div style={{ width:"100px"}}> Variable </div> | <div style={{ width:"200px"}}> Description </div>| <div style={{ width:"135px"}}> How To Access </div> |
 | :---------- | :---------- | :----------|
-| selected | If the **enable multiple selection** option is turned off, then the variable is an array of objects, and the first object holds the value of the selected button. However, if it is turned on, the variable type changes from an array to an object, and the selected button values are stored as a string within that object. | Access the value using `{{components.buttongroup1.selected[0]}}` or `{{components.buttongroup1.selected}}` |
+| selected | If the **enable multiple selection** option is turned off, then the variable is an array of objects, and the first object holds the value of the selected button. However, if it is turned on, the variable type changes from an array to an object, and the selected button values are stored as a string within that object. | Access the value using `{{components.buttongroup1.selected[0]}}` or `{{components.buttongroup1.selected}}`. |
```

Comment:
Here as well, you need to update the content of description and how to access column. ([reference](https://docs.tooljet.com/docs/widgets/text-input/#exposed-variables) ( Please just check the sentence structure from the given reference ))